### PR TITLE
add new term MONDO:0100124 NAA10-related syndrome

### LINKED ISSUE #1512: Need New MONDO ID for NAA10-related syndrome
Preferred term label: NAA10-related syndrome

Synonyms/Other Disease Assertions: Odgen Syndrome, Lenz Microphthalmia Syndrome, Autism Spectrum Disorder, Intellectual Disability

Definition (free text, please give PubMed ID, if applicable, in format PMID:#######): Variants in NAA10 are currently associated with two diseases in Monarch: Odgen Syndrome and Lenz Microphthamia Syndrome. It has also been associated with non-syndromic Intellectual Disability.  The ID/Autism experts have reviewed the disease assertions/available literature and have determined that evidence suggests that patients that have variants in the NAA10 gene demonstrate symptoms such as developmental delay, ID, ASD, hypotonia, facial dysmorphism, cardiac anomalies, and/or skeletal anomalies. The ID/Autism GCEP suggests that the term "NAA10-related syndrome" is more suited to the symptoms displayed in patients with variants in NAA10. (PMIDs: 29558889, 31127942, 28967461, 31174490, 31088393, 29748569, 26522270, 24431331, 23871722, 27094817, 25099252)
 

Parent term (use [OLS](https://www.ebi.ac.uk/ols/ontologies/mondo), or your favorite ontology browser): X-Linked Syndromic Intellectual Disability

Your nano-attribution: ClinGen ID/Autism Gene Curation Expert Panel

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
closes #1512

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -432964,6 +432967,23 @@ intersection_of: MONDO:0005550 ! infectious disease
 intersection_of: realized_in_response_to_stimulus NCBITaxon:1335626 ! Middle East respiratory syndrome-related coronavirus
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100124
+name: NAA10-related syndrome
+def: "Any X-linked syndromic intellectual disability in which the cause of the disease is a mutation in the NAA10 gene. Patients with variants in the NAA10 gene demonstrate symptoms such as developmental delay, intellectual disability, autism spectrum disorder, hypotonia, facial dysmorphism, cardiac anomalies, and/or skeletal anomalies." [https://clinicalgenome.org/affiliation/40006/, PMID:23871722, PMID:24431331, PMID:25099252, PMID:26522270, PMID:27094817, PMID:28967461, PMID:29558889, PMID:29748569, PMID:31088393, PMID:31127942, PMID:31174490]
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:432973]: Def should match name.

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo]: I was following the pattern for the definition:
https://github.com/monarch-initiative/mondo/blob/master/src/patterns/disease_series_by_gene.yaml

The parent of this class is 'X-linked syndromic intellectual disability'

Should the definition read: "NAA10-related syndrome is an X-linked syndromic intellectual disability in which..."?

I will revise this.

**Comment by nicolevasilevsky** [comment]
- [ ] fix conflict, then I can merge

**Comment by nicolevasilevsky** [comment]
the build passed

**Pull request merged**